src/audit.multiple.md

```diff
@@ -0,0 +1,88 @@
+## Multiple Roles or Resources [/api/audit/events{?role%5B%5D,resource%5B%5D,event_action,kind,limit,till,since}]
```

Comment:
Since this is an existing route, shouldn't this doc enhance the description of that route?